examples.md

```diff
@@ -170,6 +180,33 @@ The yarn cache directory will depend on your operating system and version of `ya
       ${{ runner.os }}-yarn-
 ```
 
+## OCaml/Reason - esy
+Esy allows you to export built dependencies and import pre-built dependencies.
+```yaml
+    - name: Restore Cache
+      id: restore-cache
+      uses: actions/cache@v1
+      with:
+        path: _export
+        key:  ${{ runner.os }}-esy-${{ hashFiles('esy.lock/index.json') }}
+        restore-keys: |
+          ${{ runner.os }}-esy-
+    - name: Esy install
+      run: 'esy install'
+    - name: Import Cache
+      run: |
+        esy import-dependencies _export
+        rm -rf _export
```

Comment:
What's the benefit of deleting `_export`?

Reply:
Two reasons:
- When you're reexporting, if a dependency has changed, if you didn't erase the folder before, you'll end up with both archives. This is not an "issue" because it won't create a failure, but you will have unwanted elements that can be quite heavy in your cache.
- It cleans your folder, if ever your project creates another `_export` folder it might clash (although the chances someone does this are low

That being said, it is not mandatory and can be removed. It could also be put right before "export-depencencies" which therefore will be executed just in case of cache miss. But this `rm -rf _export` is not really expensive in terms of computation.